f2/src/__stories__/Forms.stories.mdx

```diff
@@ -143,6 +143,13 @@ View [Field](/?path=/docs/components-field--input-field) documentation for more
 
 View [FormArrayControl](/?path=/docs/components-form-array-control--default-story) documentation for more information on how to place radio and checkbox lists in a Form.
 
+## Default Values
+
+formDefaults is gaint Javascript object contains a list of all fields from report with empty values asigned for each field.
+The value can be a empty string or empty array based on the type of field.
```

Comment:
Can you provide examples of types of field, and how they are implemented in formDefaults? You could put in a small js code block as an example.

Reply:
fixed